lib/bolt/applicator.rb

```diff
@@ -138,6 +144,20 @@ def compile(target, scope)
       result
     end
 
+    # Handles logging Puppet warnings, some of which are suppressable.
+    #
+    # @param target [Bolt::Target] The target the apply ran on.
+    # @param message [String] The log message.
+    #
+    private def handle_warning(target, message)
+      if message.include?(Puppet::Pops::Issues::RT_NO_STORECONFIGS_EXPORT.format) ||
```

Comment:
This could use a comment about what these messages actually are/why they happen.

Reply:
Commented. 👍